### PR TITLE
Added minutes

### PR DESCRIPTION
**Pull request by maximilien**
* minutes
* list of attendees
* action items

### CODE REVIEW

**Comment by maximilien** [comment]
Please review ASAP @ryjones @thb-sb @Naomi-Wash and merge

Will share in my email to mailing list to announce next meeting and agenda

Thanks

**Comment by ryjones** [comment]
now that I see what you want to do, I'll wait for a vote to approve from a TAC member

File: meeting-minutes/2024/2024-04-03-TAC.md [hunk 1]
```diff
@@ -41,3 +80,7 @@ nav_exclude: true
 * [ ] ~~Sophie Schmieg, Google~~
 * [x] Brian Jarvis, Amazon Web Services Inc.
 * [x] Thomas Bailleux, SandboxAQ
+* [x] Douglas Stebila, U. Waterloo
+* [x] Thomas Ashley, Cisco
```
Comment (ashman-p) [inline meeting-minutes/2024/2024-04-03-TAC.md]:
```suggestion
* [x] Norman Ashley, Cisco
```

**Review by ashman-p — CHANGES_REQUESTED** [review]

**Review by brian-jarvis-aws — APPROVED** [review]

**Review by ashman-p — APPROVED** [review]

File: meeting-minutes/2024/2024-04-03-TAC.md [hunk 2]
```diff
@@ -5,25 +5,64 @@ parent: Meeting Minutes
...
+  b. Max nominated Brian Jarvis (AWS)
+  c. Others can self-nominate or nominate others
+  d. Ry will set voting to decide vice-chair before next meeting
+* Logistics
+  a. Use GH (like Hyperledger) for agenda and minutes
+     i. Ry proposed process and created entry for first meeting
+  b. Mailing list for announce of agenda and minutes 
+  c. Use GH discussions for project and discord for 
+* Hackathon results
+  a. Nigel updates [link to PDF presentation]
```
Comment (planetf1) [inline meeting-minutes/2024/2024-04-03-TAC.md]:
```suggestion
  a. Nigel updates [link to PDF presentation](https://hackathon.pqcodepackage.org/202404/summary.html)
```

File: meeting-minutes/2024/2024-04-03-TAC.md [hunk 3]
```diff
@@ -41,3 +80,7 @@ nav_exclude: true
 * [ ] ~~Sophie Schmieg, Google~~
 * [x] Brian Jarvis, Amazon Web Services Inc.
 * [x] Thomas Bailleux, SandboxAQ
+* [x] Douglas Stebila, U. Waterloo
+* [x] Norman Ashley, Cisco
+* [x] Ry Jones, LF
+* [x] Hart Montgomery, LF
```
Comment (planetf1) [inline meeting-minutes/2024/2024-04-03-TAC.md]: Do all attendees get recorded, in addition to voting members.

Maybe
```suggestion
* [x] Hart Montgomery, LF
* [x] Nigel Jones, IBM [**non-voting attendee**]
```

File: meeting-minutes/2024/2024-04-03-TAC.md [hunk 4]
```diff
@@ -5,25 +5,64 @@ parent: Meeting Minutes
...
+* Need to select a vice-chair
+  a. Agreed to move to mailing list and elections
+  b. Max nominated Brian Jarvis (AWS)
+  c. Others can self-nominate or nominate others
+  d. Ry will set voting to decide vice-chair before next meeting
+* Logistics
+  a. Use GH (like Hyperledger) for agenda and minutes
+     i. Ry proposed process and created entry for first meeting
+  b. Mailing list for announce of agenda and minutes 
+  c. Use GH discussions for project and discord for
```
Comment (planetf1) [inline meeting-minutes/2024/2024-04-03-TAC.md:38]: I think we also discussed using the mailing list calendar for meeting invites (ical feed). Perhaps I misremembered.

File: meeting-minutes/2024/2024-04-03-TAC.md [hunk 5]
```diff
@@ -5,25 +5,64 @@ parent: Meeting Minutes
...
+  d. Perhaps two separate meetings
+* Working groups
+  a. OQS already started and doing well
+  b. For PQ code package the following projects
+    i. Algos
+    ii. Tooling and misc
+    iii. Education / Website / Docs
+  c. Security governance (as proposed by Emily)
+* Project lifecycle [Hart]
+  a. Hart shared template lifecycle that takes from Hyperledger and other projects [link to doc]
```
Comment (planetf1) [inline meeting-minutes/2024/2024-04-03-TAC.md]:
```suggestion
  a. Hart shared template lifecycle that takes from Hyperledger and other projects  [link to doc](https://docs.google.com/document/d/1NV-0vNgXWdc81oqT0jv0C-9Funb8dySS06u90ghF-X4/edit?usp=sharing)
```

File: meeting-minutes/2024/2024-04-03-TAC.md [hunk 6]
```diff
@@ -5,25 +5,64 @@ parent: Meeting Minutes
...
-</mark>
+# Action Items
 
-# Project Proposals 
-<mark>_include any project proposals that have been submitted and need discussion or decision._
-</mark>
+* [x] Email mailing list to elect vice-chair [Max]
+* [x] Document the minutes into GH repo and get approval and share in mailing list [Max]
+* [ ] Schedule next hackathon meeting [Nigel]
+* [ ] Propose the 3 new working groups:  Algos, Tooling / Misc, and Docs / Education / Website as sissues [Max]
```
Comment (planetf1) [inline meeting-minutes/2024/2024-04-03-TAC.md]:
```suggestion
* [ ] Propose the 3 new working groups:  Algos, Tooling / Misc, and Docs / Education / Website as issues [Max]
```

**Review by planetf1 — CHANGES_REQUESTED** [review]
Added some updates/corrections.

File: meeting-minutes/2024/2024-04-03-TAC.md (hunk 3 above)
Comment (ryjones) [reply meeting-minutes/2024/2024-04-03-TAC.md]: @planetf1 I only record the voting members.

File: meeting-minutes/2024/2024-04-03-TAC.md (hunk 4 above)
Comment (ryjones) [reply meeting-minutes/2024/2024-04-03-TAC.md:38]: working on it

**Comment by ryjones** [comment]
@planetf1 please merge if you agree

**Comment by planetf1** [comment]
An additional observation.

We have a bulleted list of attendees - I presume these are the TAC members

How about having this in two parts ie

## TAC members

- [ ] A tac member

## Additional attendees 

- [ ] A non member 

(For the record I attended, but am not a tac member)

**Comment by ryjones** [comment]
> An additional observation.
> 
> We have a bulleted list of attendees - I presume these are the TAC members
> 

Yes. Usually only track voting members. I have no preference about tracking non-voting attendees - if that's what the project wants, it's fine by me

**Review by ghost — APPROVED** [review]

**Pull request merged**